src/lib/viewers/box3d/video360/Video360Viewer.js

```diff
@@ -53,9 +53,26 @@ class Video360Viewer extends DashViewer {
         this.wrapperEl.classList.add(CSS_CLASS_VIDEO_360);
     }
 
+    /**
+     * Adds event listeners to the canvas element.
+     * Makes changes to the media controls.
+     *
+     * @override
+     * @return {void}
+     */
+    addEventListenersForMediaElement() {
+        super.addEventListenersForMediaElement();
+
+        this.wrapperEl.addEventListener('mousemove', this.mousemoveHandler);
```

Comment:
Would this add the listener to the wrapper and the medial el inside dashviewer.js since we make the super call above?

Reply:
Correct. I'm just overriding here, instead of doing an unbind, since there's additional functionality around creating the mouse move function, and throttling it. Suggestions?

Reply:
I've got a change to this, incoming. It even organizes things a little better